### PR TITLE
Feat/sui bundler markdown url

### PR DESCRIPTION
**Pull request by davecarter**
## Description
- [x] In order to minify url mismatches loading the `manifestURL` we need to simplify the RegEx.
- [x] When getting the Manifest we are moving to this object instead of a single string without breaking backwards compatibility:
```
  "sui-bundler": {
    "externals-manifest": {
         "preproduction": "https://scm-frontend-pre.MY-SITE.com/",
         "production": "https://scm-frontend-pro.MY-SITE.com/"
    }
}
```
Please review

### CODE REVIEW

File: packages/sui-bundler/loaders/ExternalsManifestLoader.js [hunk 1]
```diff
@@ -34,13 +34,16 @@ async function externalsManifestLoader(source) {
...
+      typeof manifestURL === 'string'
+        ? manifestURL
+        : manifestURL[process.env.NODE_ENV]
+    )
     const entries = Object.entries(manifest)
     const nextSource = entries.reduce((acc, entry) => {
       const [dest, src] = entry
-      const regex = new RegExp(`${dirname}${dest}`, 'g')
-      return acc.replace(regex, `${dirname}${src}`)
+      const regex = new RegExp(`${dest}`, 'g')
```
Comment (midudev) [inline packages/sui-bundler/loaders/ExternalsManifestLoader.js]: The template string doesn't make sense anymore. Neither on the L46.

Comment (davecarter) [reply packages/sui-bundler/loaders/ExternalsManifestLoader.js]: I thought that the RegEx needs to have an string...

Comment (midudev) [reply packages/sui-bundler/loaders/ExternalsManifestLoader.js]: And what's the `dest` and `src` type?

Comment (davecarter) [reply packages/sui-bundler/loaders/ExternalsManifestLoader.js]: ¯\_(ツ)_/¯  .. oks 😅

**Pull request merged**